RELEASE/scripts/autoscend/quests/level_09.ash

```diff
@@ -153,11 +153,9 @@ boolean L9_chasmBuild()
 
 
 	auto_log_info("Chasm time", "blue");
-
-	if(item_amount($item[fancy oil painting]) > 0)
-	{
-		visit_url("place.php?whichplace=orc_chasm&action=bridge"+(to_int(get_property("chasmBridgeProgress"))));
-	}
+	
+	// make sure our progress count is correct before we do anything.
+	visit_url("place.php?whichplace=orc_chasm&action=bridge"+(to_int(get_property("chasmBridgeProgress"))));
```

Comment:
This fixes an issue where it would burn a turn unnecessarily against a  Smut Orc if you got enough parts from Blech House to finish the bridge and is probably a good idea generally to make sure progress is up to date before we decide to adventure.